Automation/azKeyVault/deployment.ps1

```diff
@@ -0,0 +1,67 @@
+<#
+ .SYNOPSIS
+    Deploys a template to Azure
+ .DESCRIPTION
+    Deploys an Azure Resource Manager template
+ .PARAMETER subscriptionId
+    The subscription id where the template will be deployed.
+ .PARAMETER resourceGroupName
+    The resource group where the template will be deployed. Can be the name of an existing or a new resource group.
+ .PARAMETER templateFilePath
+    Optional, path to the template file. Defaults to template.json.
+ .PARAMETER parametersFilePath
+    Optional, path to the parameters file. Defaults to parameters.json. If file is not found, will prompt for parameter values based on template.
+#>
+
+param(
+   [Parameter(Mandatory = $True)]
+   [string]
+   $subscriptionId,
+
+   [Parameter(Mandatory = $True)]
+   [string]
+   $resourceGroupName,
+
+   [string]
+   $deploymentName = "AzureKeyVaultDeploymentThursday",
+
+   [string]
+   $templateFilePath = "./template.json",
+
+   [string]
+   $parametersFilePath = "./template.parameters.json"
+)
+
+#******************************************************************************
+# Script body
+# Execution begins here
+#******************************************************************************
+
+$ErrorActionPreference = "Stop"
+
+# sign in
+Write-Host "Logging in...";
+Connect-AzAccount;
```

Comment:
This will show a browser. The requirement was to create an unattended script (no user interaction is required)

Reply:
Yes, I had an issue using Larissa's Import-Module Login last time, but I was able to make it work this time. This will be updated on Sprint 4. Thank you.